### ISSUE TITLE
Update to newest version of flocking...

### ISSUE THREAD

**Issue by the-t-in-rtf** [colinbdclark/flocking-midi-router#1]
Since the merge of flocking-midi-improvements into master, it seems like one of the branches required to install flocking-midi-router is no longer available:

```
Tonys-MacBook-Pro:flocking-midi-router duhrer$ rm -rf node_modules && npm install
npm ERR! git rev-list -n1 gh-6: fatal: ambiguous argument 'gh-6': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 gh-6: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 gh-6: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 gh-6: 
npm ERR! git rev-list -n1 gh-6: fatal: ambiguous argument 'gh-6': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 gh-6: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 gh-6: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 gh-6: 
npm ERR! git rev-list -n1 gh-6: fatal: ambiguous argument 'gh-6': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 gh-6: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 gh-6: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 gh-6: 
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node6-lts/6.9.1/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.9
npm ERR! code 128
```

I am happy to update this to a newer hash and submit a PR, but it would be nice if you could fluid-publish a dev release instead, @colinbdclark.  It would be even nicer if you cut at least a 0.1.0 release.. ;)

**Comment by colinbdclark** [comment]
Hi @the-t-in-rtf, sorry for this stupid mistake. It's a great example of why I should never pin dependencies to a branch in <code>package.json</code>. In this case, Flocking was depending on a branch of codemirror-infusion, and when I cut the 2.2.0 release of the latter, I decided to be keen and tidy up old branches. Hence the error you saw.

The old midi-improvements branch of Flocking has been merged into master now. Probably safest to pin the dependency to <code>colinbdclark/Flocking#18e94e6fb7a51f0b37c8e69063eb00ff6d1badce</code> for now.

Flocking 0.2.0 is coming soon, hopefully, at which point I will publish regular dev releases in addition to much more frequent official releases. In the meantime, we'll suffer a bit. :(
